pkg/apis/playlist/v1/register.go

```diff
@@ -4,13 +4,32 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/apiserver/pkg/registry/rest"
+	genericapiserver "k8s.io/apiserver/pkg/server"
 )
 
 // GroupName is the group name for this API.
-const GroupName = "playlist.grafana.io"
+const GroupName = "playlist.x.grafana.com"
+const VersionID = "v0-alpha" //
+const APIVersion = GroupName + "/" + VersionID
+
+func GetAPIGroupInfo(
```

Comment:
Trying to have a single place that gets the config so that it is not split across the server and the group info